libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -245,7 +245,8 @@ class NavEKF3_core : public NavEKF_core_common
     // The sign convention is that a RH physical rotation of the sensor about an axis produces both a positive flow and gyro rate
     // msecFlowMeas is the scheduler time in msec when the optical flow data was received from the sensor.
     // posOffset is the XYZ flow sensor position in the body frame in m
-    void writeOptFlowMeas(const uint8_t rawFlowQuality, const Vector2f &rawFlowRates, const Vector2f &rawGyroRates, const uint32_t msecFlowMeas, const Vector3f &posOffset);
+    // height is the (constant) height of the sensor above the ground in m. Only used for rover
+    void writeOptFlowMeas(const uint8_t rawFlowQuality, const Vector2f &rawFlowRates, const Vector2f &rawGyroRates, const uint32_t msecFlowMeas, const Vector3f &posOffset, AP_Float height);
```

Comment:
I have never seen an `AP_Float` being passed around like this before

Reply:
Yeah ... it's a messy way. Would it be better if the EKF directed accessed the ``_height`` from the AP_Flow singleton?